pootle/apps/pootle_store/models.py

```diff
@@ -1972,7 +1971,7 @@ def update(self, overwrite=False, store=None, only_newer=False, user=None,
                                                        update_unitids, user,
                                                        revision=update_revision,
                                                        submission_type=submission_type)
-            self.file_mtime = disk_mtime
+            self.file_mtime = self.get_file_mtime() or self.file_mtime
```

Comment:
if im honest im not sure what the best thing to set it to is.

If the file has gone away/is not there then perhaps we should set it to datetime_min - atm it just leaves it the same as it is.